### ISSUE TITLE
No Board. Can't create a New Board.

### ISSUE THREAD

**Issue by Naewyrd** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#806]
**Describe the bug**
Originally installed TaleSpire on laptop. Then decided to install on other computer for more screen space. Started TaleSpire on new installation. Selected the Game I had previously created. Immediately told that I am in "The Place Between Boards". Try to get the board to load. No success. Try creating a new board. The screen gets stuck "Waiting for the Adventure to Begin".  Waited 30 minutes. Nothing happens.

Had to use Task Manager to shut TaleSpire down. Tried restarting. Deleted old board. Tried creating a new board. Same issue. Uninstalled TaleSpire. Reinstalled TaleSpire. Created new Games. Same issue every time ending with me using Task Manager to quit the application.

**Screenshots/Videos/Twitch Clips**
<img width="1552" alt="Screen Shot 2021-04-17 at 23 16 34" src="https://user-images.githubusercontent.com/82734669/115128283-9369ce00-9fd4-11eb-9abf-05be776a5e98.png">

<img width="1546" alt="Screen Shot 2021-04-17 at 23 17 11" src="https://user-images.githubusercontent.com/82734669/115128289-9d8bcc80-9fd4-11eb-8737-31ecc862ce80.png">

**Comment by LuPro** [comment]
Out of curiosity, does it still work on the laptop, or is the issue with the campaign now? And do other campaigns work on the computer, or is it broken for all campaigns/boards?

**Comment by Naewyrd** [comment]
Have since found out that the "PC" I tried this on is in fact a Mac using Parallels. Googled the issue and have seen your reply to another querent with the same issue. My laptop is a native Windows 10 PC whereas my friend's computer is merely running Windows 10 virtually.

I created new boards on my laptop with no issue and they appear on the Mac/Parallels version but are not accessible/do not render. As you said in your other response, TaleSpire for Mac/Linux is not going to happen anytime soon, virtually or otherwise. I'll just go back to my laptop. Appreciate the quick response.
